ydb/library/yql/providers/s3/provider/yql_s3_dq_integration.cpp

```diff
@@ -383,6 +405,107 @@ class TS3DqIntegration: public TDqIntegrationBase {
                 srcDesc.MutableSettings()->insert({"addPathIndex", "true"});
             }
 
+            auto useRuntimeListing = State_->Configuration->UseRuntimeListing.Get().GetOrElse(false);
+            srcDesc.SetUseRuntimeListing(useRuntimeListing);
+
+            auto fileQueueBatchSizeLimit = State_->Configuration->FileQueueBatchSizeLimit.Get().GetOrElse(1'000'000);
```

Comment:
Could we use 1000000 instead of 1'000'000? it's difficult to grep this kind of consts